backend/community/requirements.txt

```diff
@@ -0,0 +1,23 @@
+amqp==2.6.1
+asgiref==3.2.10
+billiard==3.6.3.0
+celery==4.4.7
+cycler==0.10.0
+Django==3.1.1
+django-filter==2.3.0
+django-rest-passwordreset==1.1.0
+djangorestframework==3.11.1
+djangorestframework-simplejwt==4.4.0
+drf-base64==2.0
+kiwisolver==1.2.0
+kombu==4.6.11
+numpy==1.18.2
```

Comment:
I don't think we need directly or indirectly `numpy` anywhere. Do we?

Reply:
We need `numpy` and `scipy` indirectly from solver. At least it's defined there in requirements.txt.

Reply:
Ah, ok.